labgrid/remote/client.py

```diff
@@ -1795,6 +1794,15 @@ def main():
         try:
             signal.signal(signal.SIGTERM, lambda *_: sys.exit(0))
 
+            if args.crossbar is None and env is not None:
+                # If no crossbar URL provided, try config file crossbar_url
+                args.crossbar = env.config.get_option('crossbar_url', None)
+
+            if args.crossbar is None:
+                # then LG_CROSSBAR environment variable
+                # fallback to default
+                args.crossbar = os.environ.get("LG_CROSSBAR", "ws://127.0.0.1:20408/ws")
+
```

Comment:
Since the crossbar URL and realm can be defined in several places, we should (debug) log the URL and realm that is finally used before starting the session.

Reply:
I've added a suggestion that actually uses logging instead of printing.